src/components/Cards/components/BackCard.tsx

```diff
@@ -1,13 +1,17 @@
-type BackCardProps = {
-    cvc: string
-}
+import { useForm } from "../../../hooks/useForm"
+
+const BackCard = () => {
 
-const BackCard = ({cvc}: BackCardProps) => (
-    <div className="bg-[url('assets/images/bg-card-back.png')] bg-cover w-72 h-40 md:w-96 md:h-52 rounded-lg p-6 text-white text-xs md:text-sm tracking-widest lg:col-start-2 lg:col-end-5 lg:static absolute top-5 right-5 sm:top-9 sm:right-6">
-        <p className="h-full flex justify-end items-center pr-2.5 md:pr-5">
-            { cvc|| '000'}
-        </p>
-    </div>
-)
+    const { state } = useForm()
```

Comment:
in my opinion this custom hook is not needed that's why you created the context. You could make this directly with the context instead calling the useForm

```
const BackCard = () => {
  const { state } = useContext(FormContext);
  const { cvc } = state;

```